tests/tracer/telemetry/test_writer.py

```diff
@@ -246,7 +272,8 @@ def test_send_request_exception():
 def test_telemetry_graceful_shutdown(mock_time, mock_send_request, telemetry_writer):
     telemetry_writer.start()
     telemetry_writer.stop()
-    assert httpretty.last_request().parsed_body == _get_request_body({}, "app-closing")
+    assert len(httpretty.latest_requests()) == 2
+    assert httpretty.last_request().parsed_body == _get_request_body({}, "app-closing", 2)
```

Comment:
`telemetry_writer.start()` now sends the app-started event (assuming the process did not fork). So this test now sends a requests on ` telemetry_writer.start()` and  `telemetry_writer.stop()`.